### ISSUE TITLE
addTypeDefs crashes with argument having default enum value

### ISSUE THREAD

**Issue by vladar** [graphql-compose/graphql-compose#221]
## Steps to reproduce:

```js
  const sc = new SchemaComposer();
  sc.addTypeDefs(`
    type Image {
      random(format: ImageFormat = JPG): String
    }
      
    enum ImageFormat {
      JPG
    }
  `);
```

### Expected result:
Doesn't throw

### Actual result
Throws:
```
Error: Type with name "ImageFormat" does not exists
```

But when deleting a default value everything works as expected

First noticed in https://github.com/gatsbyjs/gatsby/issues/19210

**Comment by stefanprobst** [comment]
Note that it also works correctly as long as the enum is defined *before*.

**Comment by nodkz** [comment]
For graphql-compose v7 it's fixed.

For v6 which is used by Gatsby needs to wait a little bit more.

**Comment by nodkz** [comment]
Fixed in graphql-compose@6.3.7 and graphql-compose@7.4.3

Please reopen the issue if the error stays.
